bytebase/bytebase.com
# feat: develop seo pages

**This PR:**
Adding seo pages

**Steps to test**

- Open the [Main page](https://bytebase-git-develop-seo-pages-bytebase-pixelpoint.vercel.app/)
- Open all pages from DATABASES and INTEGRATION footer sections; 
- Confirm that everything looks and works as expected.
